src/pages/CommitDetailPage/CommitCoverage/routes/IndirectChangesTab/IndirectChangesTable/CommitFileDiff/CommitFileDiff.test.jsx

```diff
@@ -332,10 +327,18 @@ describe('CommitFileDiff', () => {
   })
 
   describe('when there is no data', () => {
-    beforeEach(() => {
-      setup({ impactedFile: null })
+    let consoleSpy
+
+    beforeAll(() => {
+      consoleSpy = vi.spyOn(console, 'error').mockImplementation(() => {})
     })
+
+    afterAll(() => {
+      consoleSpy.mockRestore()
+    })
+
```

Comment:
are we using consoleSpy?

Reply:
Sorta, there is an expected console output (testing bad input) that doesn't matter for this test specifically, but we don't want it to clutter the test output. The other reason why we assign it is that so we can restore it after these "bad" tests so that if there are any other console outputs they will actually output to the console. Without the `mockRestore` the console would be disabled for the remainder (and potentially at any point of the test run if not ran sequentially)

Reply:
this as a comment for future readers could be useful